src/hotspot/share/gc/serial/serialArguments.hpp

```diff
@@ -32,7 +32,7 @@ class CollectedHeap;
 class SerialArguments : public GenArguments {
 private:
   virtual CollectedHeap* create_heap();
-  virtual void initialize_heap_flags_and_sizes();
+  virtual void initialize();
```

Comment:
All other GC's put the `initialize` function before the `create_heap`. It might be nice to keep that consistent.